NAMESPACE

```diff
@@ -1,10 +1,8 @@
 # Generated by roxygen2: do not edit by hand
 
 export(hello_admiral)
+import(admiraldev)
```

Comment:
ummm...do we want to have the whole package or just bits and bobs??

Reply:
I think it could go either way, in `admiral` we import the whole thing, in something like `admiralvaccine` they use bits and blobs, went ahead and copied the bits and blobs over to demonstrate?